config/cime/config_compilers.xml

```diff
@@ -19,7 +19,21 @@
      </SLIBS>  
      <SCC MPILIB="mpi-serial">gcc</SCC>
      <SFC MPILIB="mpi-serial">gfortran</SFC>
-   </compiler>
+</compiler>
+   
+<compiler MACH="container">
+  <CPPDEFS>
+    <!-- these flags enable nano timers -->
+    <append MODEL="gptl"> -DHAVE_NANOTIME -DBIT64 -DHAVE_VPRINTF -DHAVE_BACKTRACE -DHAVE_SLASHPROC -DHAVE_COMM_F2C -DHAVE_TIMES -DHAVE_GETTIMEOFDAY </append>
+  </CPPDEFS>
+  <NETCDF_PATH>/usr/local</NETCDF_PATH>
+  <PNETCDF_PATH>/usr/local</PNETCDF_PATH>
+  <LDFLAGS>
+  </LDFLAGS>
+  <SLIBS>
+    <append> -lnetcdf -lnetcdff -llapack -lblas </append>
+  </SLIBS>
+</compiler>
```

Comment:
If the compiler is set here, then should we delete https://github.com/huitang-earth/NorESM_LandSites_Platform/blob/27_NREC_VM_CentOs8_fixes/.machine/NREC_VM_setup/install_platform.sh#L46-L70 and restore https://github.com/huitang-earth/NorESM_LandSites_Platform/blob/27_NREC_VM_CentOs8_fixes/.machine/NREC_VM_setup/install_platform.sh#L76-L81 ?